SUI/SUI/View+Extention.swift

```diff
@@ -0,0 +1,29 @@
+// View+Extention.swift
```

Comment:
```suggestion
// View+Extension.swift
```
И название файлика тоже